sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -450,6 +450,21 @@ def run_pipeline(self,
         pipeline_options.DirectOptions).direct_runner_bundle_repeat
     self._num_workers = options.view_as(
         pipeline_options.DirectOptions).direct_num_workers or self._num_workers
+
+    # set direct workers running mode if it is defined with pipeline options.
+    if options.view_as(pipeline_options.DirectOptions).direct_running_mode \
+        is not None:
```

Comment:
Could this be None any more? Since it has a default value.

Reply:
Rephrased it. Thank you for catching it.